src/screens/FeedScreen/elements/BlurredLayer.js

```diff
@@ -85,11 +85,14 @@ const styles = StyleSheet.create({
     top: 0,
     zIndex: 10,
     justifyContent: 'center',
-    alignItems: 'center'
+    alignItems: 'center',
+    overflow: 'hidden'
   },
   container: {
     justifyContent: 'center',
-    alignItems: 'center'
+    alignItems: 'center',
+    width: '100%',
+    height: '100%'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `BlurredLayer.js` file seem fine. The addition of `overflow: 'hidden'` to the styles will ensure that any child elements that go outside the parent's boundary will be clipped, and won't be visible. This can be useful in certain scenarios where you want to hide parts of elements that go outside a container.

The changes to the `container` style are also good. By setting the width and height to '100%', you're ensuring that the container takes up the full width and height of its parent. This is a common practice in CSS to make sure a child element expands to fill its parent.

However, please ensure these changes do not affect other components that use this `BlurredLayer.js`, as it might change their appearance or behavior unexpectedly.
```diff
    justifyContent: 'center',
-   alignItems: 'center'
+   alignItems: 'center',
+   overflow: 'hidden'
  },
  container: {
    justifyContent: 'center',
    alignItems: 'center',
+   width: '100%',
+   height: '100%'
  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->